pyfolio/utils.py

```diff
@@ -622,3 +623,9 @@ def set_legend_location(ax):
 
     ax.legend(frameon=True, framealpha=0.5, loc='upper left',
               bbox_to_anchor=(1.05, 1))
+
+    color_cycle = cycler('color', COLORS)
+    ax.set_prop_cycle(color_cycle)
```

Comment:
Seems like we might be able to do this in one line, without importing cycler? Like:
```python
ax.set_prop_cycle('color', COLORS)
```
https://matplotlib.org/api/_as_gen/matplotlib.axes.Axes.set_prop_cycle.html

Reply:
fixed